Source/WebCore/CMakeLists.txt

```diff
@@ -1249,6 +1249,7 @@ set(WebCore_NON_SVG_IDL_FILES
     page/UndoItem.idl
     page/UndoManager.idl
     page/VisualViewport.idl
+    page/VkConsts.idl
```

Comment:
In context of upstreaming we should enable this interface based on build switch, eg: ENABLE_OIPF_VK

Reply:
ENABLE_OIPF_VK compilation flag added